strongbox-aql/src/main/groovy/aql-dynamic-grammar.groovy

```diff
@@ -0,0 +1,23 @@
+import org.carlspring.strongbox.artifact.coordinates.ArtifactLayoutLocator;
+import org.jtwig.JtwigTemplate;
+import org.jtwig.JtwigModel;
+
+
+def layoutMap = ArtifactLayoutLocator.getLayoutEntityMap();
+
+layoutMap.each{ k, v -> println "${v.getArtifactCoordinates()}" };
+
+JtwigTemplate template = JtwigTemplate.fileTemplate("$project.basedir/src/main/twig/AQL.g4.twig");
+JtwigModel model = JtwigModel.newModel().with("layoutMap", layoutMap);
+
+new File("$project.basedir/target/antlr4").mkdirs();
```

Comment:
I think that the variable here should be replaced with `${project.basedir}`.

Reply:
sorry, can you please clarify? it's already `$project.basedir`

Reply:
It's missing the curly braces.

Reply:
it's assumed to be like this. `gmaven-plugin` expose this variables here.

Reply:
I see. Okay. Thanks for the clarification!